include/robot_agent.hpp

```diff
@@ -35,6 +39,8 @@ class RobotAgent
     const int control_port_;
     const int feedback_freq_hz_;
```

Comment:
Did you add `feedback_freq_hz_` to the config file?

Reply:
added